R/map.R

```diff
@@ -48,9 +48,12 @@ Map <- R6Class(
   )
 )
 
+#' @export
```

Comment:
Is there a particular reason to add these exports, and in this PR? The `Map` class is intended only for internal use in Shiny, so there shouldn't be a need to add these exports.

Reply:
These are required, even for internal S3 methods. roxygen2 recently started warning users when S3 methods are missing this directive. This doesn't make the methods publicly available, it just adds an `S3method(as.list,Map)` to `NAMESPACE`, which is required by recent-ish versions of R.

Reply:
I'll move this to a new PR

Reply:
https://github.com/rstudio/shiny/pull/3988